epochStart/bootstrap/storageProcess_test.go

```diff
@@ -98,17 +101,9 @@ func TestNewStorageEpochStartBootstrap_ShouldWork(t *testing.T) {
 
 		coreComp, cryptoComp := createComponentsForEpochStart()
 		args := createMockStorageEpochStartBootstrapArgs(coreComp, cryptoComp)
-		sesb, err := NewStorageEpochStartBootstrap(args)
-		assert.False(t, check.IfNil(sesb))
-		assert.Nil(t, err)
-	})
-
-	t.Run("should work for sovereign", func(t *testing.T) {
```

Comment:
This test for sovereign was deleted, but I think we can still reuse the sovereign chain creation test in a different file